pages/rops/components/tabs.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Link, Snippet, Tabs } from '@asl/components';
+
+export default function RopsTabs({ active = 0 }) {
+
+  return (
+    <Tabs active={active}>
+      <Link page="ropsReporting.summary" label={<Snippet>tabs.summary</Snippet>} />
+      <Link page="ropsReporting.summary" label={<Snippet>tabs.establishments</Snippet>} />
```

Comment:
`ropsReporting.summary` Is this because we haven't got an establishments page yet?

Reply:
Yes, exactly that. Adding it was out of scope.